modules/core/src/test/scala/org/scalasteward/core/repoconfig/CommitsConfigTest.scala

```diff
@@ -0,0 +1,20 @@
+package org.scalasteward.core.repoconfig
+
+import cats.syntax.semigroup._
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+class CommitsConfigTest extends AnyFunSuite with Matchers {
+  test("CommitsConfig: semigroup") {
+    val emptyCfg = CommitsConfig()
+    emptyCfg |+| emptyCfg shouldBe emptyCfg
+
+    val cfg = CommitsConfig(Some("message"))
+    cfg |+| emptyCfg shouldBe cfg
+    emptyCfg |+| cfg shouldBe cfg
+
+    val cfg2 = CommitsConfig(Some("message2"))
+    cfg |+| cfg2 shouldBe cfg
+    cfg2 |+| cfg shouldBe cfg2
+  }
```

Comment:
In a later PR we could replace this whole test with cats-laws' `MonoidTests` like this:
`checkAll("Monoid[CommitsConfig]", MonoidTests[CommitsConfig].monoid)`